packages/react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -275,6 +275,9 @@ export type IDetailsRowStyleProps = Required<Pick<IDetailsRowProps, 'theme'>> &
 
   /** Whether to animate updates */
   enableUpdateAnimations?: boolean;
+
+  /** Defines the selection mode of list among single, multi or none */
+  selectionMode: any;
```

Comment:
If we make this change at all, this should be `selectionMode?: SelectionMode`

Reply:
okay got it!